google-platforms-case-study.md

```diff
@@ -0,0 +1,185 @@
+---
+title: Google platforms Case Study
+keywords: development platforms, google analytics, analytics, GA4, GTM, tracking, google tag manager
+tags: Development Platforms, google platforms, analytics
+sidebar: development-platforms
+permalink: development-platforms/google-platforms.html
+folder: development-platforms
+---
+
+## Introduction
+
+This study mainly covers Google analytics, but will also touch on Google tag manager, as it is a very related platform.
+
+Google Analytics is an analysis tool, which is used to gain insights and provide reports on user behavior on websites and/or web applications. Since Universal google analytics(UA) expires in July 2023 (Weber, 2023), this study will cover the already exciting successor google analytics 4(GA4).
+
+Google tag manager is, as the name suggests, a tag management tool. The purpose is to make it easy to manage tracking codes for various platforms, not only google analytics but also platforms such as google ads, Meta platforms/facebook pixel etc. (Fedorovicius, 2022)
```

Comment:
What is a tag?

Reply:
slight adjustement on the section and added a section with a more detailed description on a tag.

```
Google tag manager (GTM) is, as the name suggests, a tag management tool, better known as tag management system (TMS). The purpose is to make it easy to manage tracking codes for various platforms, not only google analytics but also platforms such as google ads, Meta platforms/facebook pixel etc. (Fedorovicius, 2022)

The purpose of a TMS like GTM is to allow for a quick and easy way to update measurement codes and related code fragments, which collectively is known as tags, on a website or web application. This is done by applying a small segment of Tag Manager code to the project. With that code applied the developer can then safely and easily deploy analytics and measurement tag configurations from a web-based user interface. (Google Help, 2023)
```